### ISSUE TITLE
CS doesnt provide resource(conquer+embassy)

### ISSUE THREAD

**Issue by bigcat88** [LoneGazebo/Community-Patch-DLL#4827]
_1. Mod version (i.e Date - 4/23):_
10-10-2(hotfix installed on turn 0)

_2. Mod list (if using Vox Populi only, leave blank):_
Naval Warfare
Really Advanced Setup

Genoa(without any embassy) captured Prague(with my embassy established on oil) and I didn't see that oil in my resources(on last screen). I am Russia, so that 6 oil is mine.

![20181018182323_1](https://user-images.githubusercontent.com/13381981/47165617-065e3500-d303-11e8-99e6-4c766b83f7fc.jpg)

![20181018182325_1](https://user-images.githubusercontent.com/13381981/47165805-7ff62300-d303-11e8-8976-77ca9146304b.jpg)

![20181018182332_1](https://user-images.githubusercontent.com/13381981/47165809-81bfe680-d303-11e8-8971-d39910e5aff5.jpg)

**Comment by TechpriestEnginseer** [comment]
I believe the City-State itself has to discover the Oil (as seen that the UI should have shown you have unimproved Oil at your coast and at the farm)...

**Comment by bigcat88** [comment]
That oil is on land. It's different technologies and eras.

**Comment by bigcat88** [comment]
It's counted(screen number 2). Oil on land 3 + 6 oil on sea.

**Comment by TechpriestEnginseer** [comment]
Yes, but they still need to discover the resource to give it to you. Nonetheless, I'll look if Embassy auto-improve Oil.

**Comment by TechpriestEnginseer** [comment]
No they don't. Interesting.
